src/main/webapp/WEB-INF/templates/search.jsp

```diff
@@ -22,7 +22,9 @@
             <a href="${knowledgeCard.getLink()}" title="card-source"class="card-title">
               ${knowledgeCard.getTitle()}
             </a>
-            <code>${knowledgeCard.getCode()}</code>
+            <c:if test="${knowledgeCard.getCode() != null}">
+              <code>${knowledgeCard.getCode()}</code>
+            </c:if>
             <span class="card-description">Description:</span>
```

Comment:
Turns out removing this `<code></code>` actually makes the description wrap with the title since it's a span and not a div

Reply:
that happens sometime, but for me it make more sense to change the tag for description to <div>